samples/simple_app/app.py

```diff
@@ -109,4 +120,3 @@ async def get_configuration_data(commons: ConfigurationDataDeps = Depends()):
     allow_headers=["*"],
     allow_methods=["OPTIONS", "GET", "POST"],
 )
-app.mount("/static", StaticFiles(directory=settings.static_dir), name="static")
```

Comment:
I got rid of that to get rid on the Jinja dependency and simplify the example, the example form now just renders a JSON